packages/zapp/console/src/components/Executions/TaskExecutionsList/TaskExecutionsListItem.tsx

```diff
@@ -45,6 +45,7 @@ export const TaskExecutionsListItem: React.FC<TaskExecutionsListItemProps> = ({
   const headerText = formatRetryAttempt(taskExecution.id.retryAttempt);
   const taskHasStarted = closure.phase >= TaskExecutionPhase.QUEUED;
 
+  const logs = 'log' in taskExecution ? [taskExecution.log] : taskExecution?.closure?.logs;
```

Comment:
yeah, had TS issue while retrieving logs in another way, and this field check was copy-pasted into ternary

Reply:
oh, ok my TS checker crashed, but after restarting it shows a union type issue I was trying to avoid by adding field check
```Property 'log' does not exist on type 'TaskExecution | MapTaskExecution'.
  Property 'log' does not exist on type 'TaskExecution'.```